docs/README.md

```diff
@@ -34,29 +34,29 @@ macOS (OS X) development environment.
 
 
 # Main features
-
-  * Support for multiple analyzers, currently
-    [_Clang Static Analyzer_](http://clang-analyzer.llvm.org/) and
-    [_Clang-Tidy_](http://clang.llvm.org/extra/clang-tidy/)
-  * Store results of multiple large-scale analysis runs efficiently, either in
-    a PostgreSQL or SQLite database
+## Command line C/C++ Analysis
+  * Executes [_Clang-Tidy_](http://clang.llvm.org/extra/clang-tidy/) and [_Clang Static Analyzer_](http://clang-analyzer.llvm.org/) with Cross-Translation Unit analysis, Statistical Analysis (when checkers are available).
+  * It can create the JSON compilation database
+  * It can automatically analyze gcc cross-compiled projects: detecting gcc compiler configuration and forming the corresponding clang invocations
+  * Incremental analysis
+  * False positive suppression
```

Comment:
Should we mention the more general review comments here which can be used to suppress false positives?